src/file-manager/fm-empty-view.c

```diff
@@ -392,14 +392,14 @@ fm_empty_view_supports_uri (const char *uri,
 
 static CajaViewInfo fm_empty_view =
 {
-    FM_EMPTY_VIEW_ID,
-    "Empty",
-    "Empty View",
-    "_Empty View",
-    "The empty view encountered an error.",
-    "Display this location with the empty view.",
-    fm_empty_view_create,
-    fm_empty_view_supports_uri
+    .id = FM_EMPTY_VIEW_ID,
+    .view_combo_label = "Empty",
+    .view_menu_label_with_mnemonic = "Empty View",
```

Comment:
`view_menu_label_with_mnemonic` should contain `_` character (accelerator), while error_label don't.
To enable text translation, text should be marked with N_(TEXT).